pkg/apiOracle/valueOracle.go

```diff
@@ -11,12 +11,14 @@ import (
 	"sync"
 	"time"
 
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
 	"github.com/pkg/errors"
 	"github.com/tellor-io/telliot/pkg/config"
 	"github.com/tellor-io/telliot/pkg/util"
 )
 
-var logger = util.NewLogger("apiOracle", "valueOracle")
+var logger = log.With(util.NewLogger(), "apiOracle", "valueOracle")
```

Comment:
There should be no global logger variables, these should be passed in the component constructors.

also lets extract the component names into constants so that can use these with the log configuration for the level
```suggestion

const ComponentName = "apiOracle"
```